app/scripts/proactive/view/WorkflowView.js

```diff
@@ -6,7 +6,7 @@ define(
         'proactive/view/TaskView',
         'proactive/view/utils/undo',
         'pnotify',
-        'pnotify.buttons'
+        'pnotify.buttons',
```

Comment:
Do we need a , here, since it is the last one?

Reply:
No, you're right, it's not needed. Thank you